readthedocs/projects/tasks.py

```diff
@@ -62,8 +62,6 @@
 
 log = logging.getLogger(__name__)
 
-HTML_ONLY = getattr(settings, 'HTML_ONLY_PROJECTS', ())
```

Comment:
I can see that we have 3 projects here, with a comment `banned`, I think we can move those to use the `skip` field on the db.

Reply:
skip will skip the build entirely, sounds like we want to disable non-html builds instead. i don't think we have a way to opt users into disabling pdf/epub from our side other than this setting right now. it could just be a feature flag or something though.

Reply:
If you check the settings from the ops repo, there is this comment `banned`, and if these projects are abusive projects, probably the best to do is ban them (skip)? otherwise, the builders would time out or kill the process when building pdf/epub anyway.

Reply:
Seems fine to me